commerce-service/src/main/resources/com/liferay/commerce/internal/upgrade/v4_0_0/dependencies/CommerceOrderUpgradeProcess.sql

```diff
@@ -0,0 +1,8 @@
+UPDATE
+	CommerceOrder
+SET
+	commerceCurrencyCode = CommerceCurrency.code_
+FROM
```

Comment:
I think `UPDATE...FROM` may not work on some databases -- see https://en.wikipedia.org/wiki/Update_(SQL):

> Some databases allow the non-standard use of the FROM clause

We should do this in a truly db-agnostic way.